app/deploy.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+RUN_DIR=$(pwd -LP)
+#the presence of a .env file overrides the production file, which overrides staging, which overrides local
+source ../../.env.local
+source ../../.env.staging
+source ../../.env.production  
+source  ../../.env
+
+
```

Comment:
@netpoe what do you think about this script? I think this should work fine

I searched a lot and found this https://github.com/rekliner/next-pm2-deploy/tree/main

Reply:
one step remaining is configuring Configure Nginx as a reverse proxy

Reply:
Also, I am not against Vercel. I still pay for deploying the preview app versions. Perhaps we should think differently regarding the automated API calls that run the game stages?

Let's study the `create.ts`   API endpoint and its subsequent timed API calls.